packages/bindings/src/types.rs

```diff
@@ -141,10 +91,37 @@ pub struct AccessGrant {
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 #[serde(rename_all = "snake_case")]
 pub enum MarkerPermission {
-    Burn,
-    Deposit,
-    Delete,
-    Grant,
-    Mint,
-    Withdraw,
+    AccessAdmin,
```

Comment:
Although, these types should work for encoding messages, they will break on query - See https://github.com/provenance-io/provenance/blob/feature/add-marker-wasm/x/marker/wasm/types.go#L50

Reply:
We should keep these as they were - ie without the `Access` prefix.

Reply:
fixed